TMessagesProj/src/main/java/org/telegram/ui/bots/BotWebViewContainer.java

```diff
@@ -97,7 +97,7 @@ public abstract class BotWebViewContainer extends FrameLayout implements Notific
     private final static int REQUEST_CODE_WEB_VIEW_FILE = 3000, REQUEST_CODE_WEB_PERMISSION = 4000, REQUEST_CODE_QR_CAMERA_PERMISSION = 5000;
     private final static int DIALOG_SEQUENTIAL_COOLDOWN_TIME = 3000;
 
-    private final static List<String> WHITELISTED_SCHEMES = Arrays.asList("http", "https");
+    private final static List<String> WHITELISTED_SCHEMES = Arrays.asList("http", "https", "wc");
```

Comment:
Actually for web3 bot devs

Reply:
@DrKLO , @Bubu , @alabiaga can you take a look please?

Reply:
Oh, really need this change. I think WS must have option